documentation/modules/security/ref-certificates-and-secrets.adoc

```diff
@@ -190,7 +190,7 @@ spec:
     # ...
 ----
 
-For more information on configuring template properties, see xref:assembly-customizing-kubernetes-resources-str[].
+For more information on configuring template properties, see xref:assembly-customizing-kubernetes-resources-str[].== Adding labels and annotations to Secrets
```

Comment:
Is this change intentional? This seems like it adds a title at the end of a sentence ...

Reply:
No it is not I'm removing it!